### PR TITLE
Pumping up the versions for allennlp, spacy and pandas

### PR DESCRIPTION
**Pull request by dcfidalgo**
This PR pumps up the version of the allennlp, spacy and pandas.
I still have to check all the compatibility for allennlp 1.2.0 -> this is a draft.

### CODE REVIEW

**Comment by dcfidalgo** [comment]
OK, i think i found a reasonable fix for the change in allennlp's `load_archive` method. Should keep backward compatibility.

File: src/biome/text/pipeline.py [hunk 1]
```diff
@@ -812,7 +812,11 @@ class _PreTrainedPipeline(Pipeline):
 
     def __init__(self, pretrained_path: str, **extra_args):
         self._binary = pretrained_path
-        archive = load_archive(self._binary, **extra_args)
+        archive = load_archive(
+            self._binary,
+            overrides={"dataset_reader": {"type": "interleaving", "readers": {}}},
```
Comment (frascuchon) [inline src/biome/text/pipeline.py:817]: Could you please insert a little comment explaining the reason of this line?

**Review by frascuchon — APPROVED** [review]
Nice !!

Just a minimal comment

File: src/biome/text/pipeline.py (hunk 1 above)
Comment (dcfidalgo) [reply src/biome/text/pipeline.py:819]: Done, totally forgot it!

**Pull request merged**